src/amdgpu.cpp

```diff
@@ -264,7 +264,12 @@ void amdgpu_get_metrics(){
 	gpu_info.MemClock = amdgpu_common_metrics.current_uclk_mhz;
 
 	// Use hwmon instead, see gpu.cpp
-	// gpu_info.CoreClock = amdgpu_common_metrics.current_gfxclk_mhz;
+	if ( deviceID == 0x1435 || deviceID == 0x163f )
+	{
+		// If we are on VANGOGH (Steam Deck), then
+		// always use use core clock from GPU metrics.
+		gpu_info.CoreClock = amdgpu_common_metrics.current_gfxclk_mhz;
```

Comment:
This will be overwritten in [gpu.cpp#138](https://github.com/flightlessmango/MangoHud/blob/f168483c3ef059abf6a5072ffbdd2c7c3104e39b/src/gpu.cpp#L141C11-L141C11)
Should also have a condition based on deviceID there

Reply:
It is based on deviceID already?

It will not be overwrittenm this comes after.

Reply:
Ah yes you are correct, looks good then!

Reply:
Ah yes you are correct, looks good then!